go/consensus/tendermint/apps/staking/state/gas.go

```diff
@@ -75,8 +76,13 @@ func AuthenticateAndPayFees(
 	if err != nil {
 		return fmt.Errorf("failed to fetch staking consensus parameters: %w", err)
 	}
-	if err = needed.Add(&params.MinTransactBalance); err != nil {
-		return fmt.Errorf("adding MinTransactBalance to fee: %w", err)
+	var minTransactBalanceExempt bool
+	if params.MinTransactBalanceExemptMethods[method] {
```

Comment:
But if there's no registry check then someone can still submit these transactions (resulting in an account being created with a nonce) and they will fail but the account will remain?